runtimes.yml

```diff
@@ -0,0 +1,59 @@
+version: 0.1
```

Comment:
This file originates from the Ubuntu Standard 6.0 runtime.yml:
https://github.com/aws/aws-codebuild-docker-images/blob/master/ubuntu/standard/6.0/runtimes.yml

I updated it to replace the Python 3.10 runtime, since we install 3.8 instead.